src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -71,7 +111,9 @@ void SimulatedTestFixture::SetUp()
 
 void SimulatedTestFixture::enableVisualizer()
 {
-    full_system_gui = std::make_shared<ThreadedFullSystemGUI>(mutable_thunderbots_config);
+    full_system_gui =
+        std::make_shared<ThreadedFullSystemGUI>(friendly_mutable_thunderbots_config);
+    // TODO: add support for enemy
```

Comment:
TODO should be attached to an issue, if it still has to be done

Reply:
@xyztiger not quite sure why we need support for enemy - viewing the GUI from just the yellow perspective gets the job done IMO.

Reply:
On my end when I run the visualizer I can't see any plays/tactics info. For this comment, I was referring to support for showing what tactics enemy robots are currently running.

Reply:
Lets make a ticket for that then and attach it to the TODO

Reply:
We decided to remove this TODO because simulated tactic tests never showed tactic info in the visualizer